internal/service/encryptionatrest/resource_test.go

```diff
@@ -107,6 +124,18 @@ func TestAccEncryptionAtRest_basicAzure(t *testing.T) {
 			TenantID:          conversion.StringPtr(os.Getenv("AZURE_TENANT_ID")),
 		}
 
+		azureKeyVaultAttrMap = map[string]string{
```

Comment:
looks like we are converting from a `admin.AzureKeyVault` to `map[string]string` in ~4 places. Maybe refactor to a helper function?

Reply:
added helper function